.eslintrc.js

```diff
@@ -1,6 +1,10 @@
 const config = {
     parser: 'babel-eslint',
-    extends: ['@magento']
+    extends: ['@magento'],
+    rules: {
+        'no-undef': 'off',
+        'no-useless-escape': 'off'
+    }
```

Comment:
Why do we have to define these two rules everywhere?

Reply:
Because we disregard them throughout the application. The alternative would be to add these rules to `magento-eslint`.

Reply:
I added these to magento-eslint and published but I'm reconsidering that... I don't think we should be disabling `no-undef`. We should be updating some config with those values somewhere.

Reply:
Created [PWA-207](https://jira.corp.magento.com/browse/PWA-207).